spec/Gaufrette/Adapter/AclAwareAmazonS3Spec.php

```diff
@@ -12,25 +12,25 @@ class AclAwareAmazonS3Spec extends ObjectBehavior
      * @param \Gaufrette\Adapter $adapter
      * @param \AmazonS3 $service
      */
-    function let(Adapter $adapter, AmazonS3 $service)
+    public function let(Adapter $adapter, AmazonS3 $service)
```

Comment:
Actually this fix is wrong IMHO: spec functions don't need `public` visibility keyword